lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
All right, what's the final suggestion then? Allow arbitrary padding via `theano_extensions`? Or remove `(0,)` or `(0, 0)` and only keep `0` in the docstring? Independently of that, I'll give a warning for `pad='valid'` in all layers.